app/Helpers/Helper.php

```diff
@@ -1317,12 +1317,94 @@ public static function getUnitConversionFactor($unit) {
 
 
     /*
-     * I know it's gauche  to return a shitty HTML string, but this is just a helper and since it will be the same every single time,
+     * I know it's gauche to return a shitty HTML string, but this is just a helper and since it will be the same every single time,
      * it seemed pretty safe to do here. Don't you judge me.
      */
     public static function showDemoModeFieldWarning() {
         if (Helper::isDemoMode()) {
             return "<p class=\"text-warning\"><i class=\"fas fa-lock\"></i>" . trans('general.feature_disabled') . "</p>";
         }
     }
+
+
+    /**
+     * Ah, legacy code.
+     *
+     * This corrects the original mistakes from 2013 where we used the wrong locale codes. Hopefully we
+     * can get rid of this in a future version, but this should at least give us the belt and suspenders we need
+     * to be sure this change is not too disruptive.
+     *
+     * In this array, we ONLY include the older languages where we weren't using the correct locale codes.
+     *
+     * @author A. Gianotto <snipe@snipe.net>
+     * @since 6.3.0
+     *
+     * @param $language_code
+     * @return string []
```

Comment:
This should be the following I think:

```suggestion
     * @return string
```

Though we technically might not return a string since the `return` is in an `if`.